bundle/bundle.go

```diff
@@ -372,30 +407,29 @@ func (r *Reader) Read() (Bundle, error) {
 	var err error
 
 	bundle.Data = map[string]interface{}{}
+	bundle.Patch = newPatch()
 
-	bundle.Signatures, descriptors, err = listSignaturesAndDescriptors(r.loader, r.skipVerify, r.sizeLimitBytes)
+	bundle.Signatures, bundle.Manifest, descriptors, err = listSignaturesAndDescriptors(r.loader, r.skipVerify, r.sizeLimitBytes)
 	if err != nil {
 		return bundle, err
 	}
 
-	err = r.checkSignaturesAndDescriptors(bundle.Signatures)
-	if err != nil {
-		return bundle, err
+	// skip JWT signature verification for delta bundles
+	if bundle.Manifest.Type != DeltaBundleType {
```

Comment:
nit: I would have probably written this the othey way (e.g., bundle.Manifest.Type == SnapshotBundleType) instead of negating everything.

Reply:
Updated to remove negated check